pyvtt/vttfile.py

```diff
@@ -24,6 +24,10 @@
 BIGGER_BOM = max(len(bom) for bom, encoding in BOMS)
 
 
+reload(sys)  
```

Comment:
This is a little bit scary, we're changing the global (sys) configuration.

We should try to find a less intrusive way of fixing this.